tests/webapps/pom.xml

```diff
@@ -0,0 +1,138 @@
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.runtimes.jetty.tests</groupId>
+    <artifactId>tests-parent</artifactId>
+   <version>0.1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>tests-webapps-parent</artifactId>
+  <name>Jetty-Runtime :: Tests :: WebApps Parent</name>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>test-war-hello</module>
+  </modules>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.eclipse.jetty.toolchain</groupId>
+        <artifactId>jetty-test-helper</artifactId>
+        <version>4.0</version>
+        <scope>test</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/docker</directory>
+        <filtering>true</filtering>
+        <targetPath>${project.build.directory}/docker</targetPath>
+      </resource>
+    </resources>
+    <plugins>
+      <plugin>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <configuration>
+          <!-- 
+             Tests cannot and should not be validated with checkstyle.
+             these tests need to be flexible enough for good, bad, quirky
+             and everything in between with regards to compliance and
+             validity.
+             Having them pass checkstyle makes them invalid for integration
+             test cases
+             -->
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>2.19.1</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.maven.surefire</groupId>
+            <artifactId>surefire-junit47</artifactId>
+            <version>2.19.1</version>
+          </dependency>
+        </dependencies>
+        <configuration>
+          <systemPropertyVariables>
+            <app.deploy.project>${app.deploy.project}</app.deploy.project>
+            <app.deploy.version>${maven.build.timestamp}</app.deploy.version>
+          </systemPropertyVariables>
+        </configuration>
+
+      </plugin>
+      <!-- This can be re-enabled when app-maven-plugin issue is fixed
```

Comment:
Can this just move into the individual test sub-modules right now? ...Oh, you already did that.